packages/twenty-front/src/modules/object-record/record-field/form-types/components/FormMultiSelectFieldInput.tsx

```diff
@@ -63,6 +64,14 @@ const StyledPlaceholder = styled.div`
   width: 100%;
 `;
 
+const safeParsedValue = (value: string) => {
```

Comment:
Looks like a duplicate for `parseJson` from `twenty-shared`, but here you return the input value instead of null. I'm fine with that but we might want to have a single util. LGTM